101-batch-with-keyvault-pfx-password/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "batch account with keyvault pfx password",
+    "description": "This template demostrates using Azure Batch service with pfx password certificate from keyvault",
+    "summary": "This will add certificate under azure batch from keyvault pfx with password",
+    "githubUsername": "manishshar",
+    "dateUpdated": "2018-10-19"
```

Comment:
update the date

Reply:
Done. updated to "dateUpdated": "2018-11-29"